packages/google_mlkit_barcode_scanning/ios/google_mlkit_barcode_scanning.podspec

```diff
@@ -15,7 +15,7 @@ Pod::Spec.new do |s|
   s.source_files = 'Classes/**/*'
   s.public_header_files = 'Classes/**/*.h'
   s.dependency 'Flutter'
-  s.dependency 'GoogleMLKit/BarcodeScanning', '~> 6.0.0'
+  s.dependency 'GoogleMLKit/BarcodeScanning', '~> 5.0.0'
```

Comment:
Likewise for iOS, it is not updated on the website
It there an alternative for this too ?

Reply:
it is not updated in the website. I pulled those by running `pod repo update` and that is the latest it pulled.